### ISSUE TITLE
uhdm.h is installed twice in different locations

### ISSUE THREAD

**Issue by hzeller** [chipsalliance/UHDM#197]
... this will possibly create confusing situations:
```
PREFIX=/tmp/uhdm-test-install make install
ls -l $(find /tmp/uhdm-test-install -name uhdm.h)
```

yields
```
-rw-r--r-- 1 user user 8130 Apr 29 16:23 /tmp/uhdm-test-install/include/uhdm/headers/uhdm.h
-rw-r--r-- 1 user user 8130 Apr 29 16:23 /tmp/uhdm-test-install/include/uhdm/uhdm.h
```

**Comment by alaindargelas** [comment]
Agreed

**Comment by hzeller** [comment]
Fixed with #488
